src/components/change-password/change-password.tsx

```diff
@@ -0,0 +1,148 @@
+import React from 'react';
+import { StyleGenerator } from '../styling/style-generator.service';
+import { OffsetStyle } from '../styling/offset-style';
+import { htmlAttributes } from 'sitefinity-react-framework/widgets/attributes';
+import { classNames } from 'sitefinity-react-framework/utils/classNames';
+import { WidgetContext } from 'sitefinity-react-framework/widgets/widget-context';
+import { PostPasswordChangeAction } from './interfaces/PostPasswordChangeAction';
+import { StylingConfig } from '../styling/styling-config';
+import { getUniqueId } from 'sitefinity-react-framework/utils/getUniqueId';
+import { RestExtensionsService } from 'sitefinity-react-framework/sdk/rest-extensions';
+import { RestSdkTypes, RestService } from 'sitefinity-react-framework/sdk/rest-service';
+import { MixedContentContext } from 'sitefinity-react-framework/widgets/entities/mixed-content-context';
+import { ChangeForm } from './change-form';
+import { VisibilityStyle } from '../styling/visibility-style';
+
+const defaultMixedContent = {
+    ItemIdsOrdered:null,
+    Content:[ {
+        Type:RestSdkTypes.Pages,
+        Variations:null
+    }]
+};
+
+export async function ChangePassword(props: WidgetContext<ChangePasswordEntity>) {
+    const entity = {
+        PostPasswordChangeRedirectPage: defaultMixedContent,
+        PostPasswordChangeMessage:  'Your password was changed successfully!',

```

Comment:
should those strings be extracted in order to be translated in other languages?

Reply:
these are the default values - they can be changed through the settings of the widget

Reply:
default values will be set in the entities when the abstraction of the designers metadata is merged